packages/client/utils/constants.ts

```diff
@@ -144,6 +144,9 @@ export const RETROSPECTIVE = 'retrospective'
 /* Retro DnD types */
 export const DISCUSSION_TOPIC = 'DISCUSSION_TOPIC'
 
-/* Spotlight Groups */
+/* Spotlight */
 export const MAX_REDUCTION_PERCENTAGE = 1
 export const MAX_RESULT_GROUP_SIZE = 10
+export const MAX_SPOTLIGHT_COLUMNS = 3
+export const SPOTLIGHT_TOP_SECTION_HEIGHT = 236
+export const SPOTLIGHT_GROUPS_PADDING = 60
```

Comment:
If a variable is used in two files, I've added it to the constants. If it's just used in one file, I think it's OK to define it locally